### DISCUSSION TITLE
0.9.5202 on Mac can't drag and drop groups

### DISCUSSION THREAD

**Discussion by Zulaque** [meerk40t/meerk40t#2717] [Q&A] [answered]
0.9.5202 on Mac can't drag and drop groups directly in to operations. After some testing I realized that I have to move each path inside the group by hand. That is weird and super slow. I'm missing something?
I'm working in 0.9.4 and there I can drag and drop complete groups without issue.
Thanks.

**Comment by jpirnay** [comment]
That was unintended and has been fixed, thanks for noticeing it. If you run from source then you can pull the code again and should be fine. If not then you need to wait for the next release - not that far away I am hoping.

**Reply by Zulaque** [reply]
Thanks for the clarification! 
BTW, is there a difference between the M1 and non-M1 versions on the download links? I'm running a MacBook Air M1.
9.4 is working fine for now, so I'll wait for the next release, no problem.
And thanks for all the hard work, MeerK40t is my daily workhorse.

**Comment by Zulaque** [comment]
Just dowloaded the last Mac version 9.5202, the issue is still present. Can't drag and drop groups. Is a BIG hassle.

**Comment by jpirnay** [comment, marked as answer]
It is fixed, but a Mac version has not yet been produced (it's a real PIA to create one take multiple steps to get a signed and approved version)
![drop](https://github.com/user-attachments/assets/976bcfdb-c63e-4aef-8dba-232fad6943f5)

**Reply by Zulaque** [reply]
Oh! Thanks for the confirmation! Looking forward to this next version. Thanks for all the great work!
